### DISCUSSION TITLE
matplotlib/PyPlot issue

### DISCUSSION THREAD

**Discussion by scott314159** [slimgroup/JUDI.jl#257] [General]
I just did a fresh install on a Linux system and got an issue:
ERROR: InitError: KeyError: key "get_cmap" not found

It turns out that matplotlib's API has been updated.  

There is a solution posted at:
https://github.com/JuliaPy/PyPlot.jl/pull/583

but I guess the solution has not been deployed yet.  So, I thought it might help someone looking in this forum about this issue since it took me a little while to track this down.

**Comment by mloubout** [comment]
Yes I ran into it as well,  tanks for referencing it here.
